benchmarks/shelley3pools/tx-generator-script.json

```diff
@@ -0,0 +1,89 @@
+[
```

Comment:
@MarcFontaine, can we improve the density, by reformatting this very JSON as one line per command?

Reply:
..unless the command itself is large, of course.

Reply:
Best would be to add a THIS IS A GENERATED FILE! DO NOT EDIT ! comment